source/opt/optimizer.cpp

```diff
@@ -116,7 +116,6 @@ Optimizer& Optimizer::RegisterPerformancePasses() {
   return RegisterPass(CreateRemoveDuplicatesPass())
       .RegisterPass(CreateMergeReturnPass())
       .RegisterPass(CreateInlineExhaustivePass())
-      .RegisterPass(CreateEliminateDeadFunctionsPass())
       .RegisterPass(CreateScalarReplacementPass())
```

Comment:
After Inlining there will be a lot of dead code because most functions will be dead.  Removing the dead functions at this point will probably help with compile time.  I don't think we should simply remove the dead function elimination.  We could run ADCE multiple times or something.

Reply:
Ok, I'll add a pass of ADCE where dead function elim used to be.